modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1119,6 +1119,42 @@ paths:
       responses:
         200:
           description: The instance started successfully
+  /fake/enum-request:
```

Comment:
I think this file should be copied rather than modified to avoid changing other test data

Reply:
It's better to make a separate file of minimal yaml?
I'm doing this because I saw [this description](https://github.com/OpenAPITools/openapi-generator/blob/master/CONTRIBUTING.md#testing): 
> For new test cases, please add to the Fake Petstore spec

Reply:
ah, ok, yeah so its fine as it is, thanks

Reply:
there is still an issue with this test, could you fix it?
https://cloud.drone.io/OpenAPITools/openapi-generator/12016/1/5